webapp/components/admin_console/admin_sidebar.jsx

```diff
@@ -528,24 +528,18 @@ export default class AdminSidebar extends React.Component {
                                     />
                                 }
                             >
-                                <AdminSidebarSection
-                                    name='storage'
-                                    title={
-                                        <FormattedMessage
-                                            id='admin.sidebar.storage'
-                                            defaultMessage='Storage'
-                                        />
-                                    }
-                                />
-                                <AdminSidebarSection
-                                    name='images'
-                                    title={
-                                        <FormattedMessage
-                                            id='admin.sidebar.images'
-                                            defaultMessage='Images'
-                                        />
-                                    }
-                                />
+                                {[
+                                    <AdminSidebarSection
+                                        key='storage'
+                                        name='storage'
+                                        title={
+                                            <FormattedMessage
+                                                id='admin.sidebar.storage'
+                                                defaultMessage='Storage'
+                                            />
+                                        }
+                                    />
+                                ]}
```

Comment:
If you change the type of the children prop for the AdminSidebarSection component to `PropTypes.node`, you won't need the array. I'd prefer to do that over needing the array and the key here